script/config_data.jl

```diff
@@ -15,7 +15,6 @@ der_use_case_map = Dict(
 )
 
 supply_choice_use_case_map = Dict(
-    "SupplyChoice" => SupplyChoice(),
```

Comment:
If we want to keep GreenDeveloper momentarily, we should also keep SupplyChoice.